src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -241,7 +241,7 @@ def default_api_version(self):
         ResourceType.MGMT_RESOURCE_SUBSCRIPTIONS: '2016-06-01',
         ResourceType.MGMT_RESOURCE_TEMPLATESPECS: '2015-01-01',
         ResourceType.MGMT_NETWORK_DNS: '2016-04-01',
-        ResourceType.MGMT_AUTHORIZATION: SDKProfile('2016-09-01', {
+        ResourceType.MGMT_AUTHORIZATION: SDKProfile('2015-07-01', {
```

Comment:
Is there a downgrade for authorization?

Reply:
This is not a downgrade, This is a bug fix. In our earlier PR we have mistakenly updated the api version @Juliehzl